src/YAxisLabels.js

```diff
@@ -26,7 +26,7 @@ function resolveYLabelsForValues(scale, values, formats, style, force = true) {
         value,
         format,
         style: _.defaults(
-          getValue(style.labelStyle, value, i),
+          getValue(style.labelStyle, { value }, i),
```

Comment:
are there no cases where `getValue` expects a string here?

Reply:
`getValue` treats all arguments after the first one as arguments that will be passed into the first argument assuming it's a function

There may have been some users that were writing their labelStyle functions thinking the argument passed to it is a string and not an object (what we saw happen to @acmei) but that won't render what is expected since the actual application of the styles via labelStyle is passed an object `{ x, y, value }` etc